specifications.yml

```diff
@@ -1007,6 +1109,123 @@ components:
         * pending -> application has started the automatic processing.
         * new -> application is new and has not yet started the automatic processing.
 
+    manualDecision:
+      type: string
+      enum:
+        - granted
+        - referred
+        - denied
+      description: |
+        Current decision of an application.
+        Possible values are
+        * denied -> application is denied and no manual review is necessary.
+        * referred -> application will go through manual review.
+        * granted -> application is granted and no manual review is necessary.
+
+    termsForCustomerApplication:
+      type: object
+      description: Terms of the application. (Look at the examples on the right to understand when they are required)
+      properties:
+        creditLimit:
+          type: number
+          description: "Credit limit for the specific customer"
+        creditLimitCurrency:
+          type: string
+          description: "Currency for the credit limit"
+        creditInsuranceLimit:
+          type: number
+          description: "Credit limit for the insurance"
+        creditInsuranceLimitCurrency:
+          type: string
+          description: "Currency for the insurance credit limit"
+
+    termsForFinancingApplication:
+      type: object
+      description: Terms of the application. (Look at the examples on the right to understand when they are required)
+      properties:
+        financeAmount:
+          type: number
+          description: Amount to be financed
+        feeSettings:
+          type: array
+          description: "List of settings for the fee"
+          items:
+            $ref: "#/components/schemas/feeSetting"
+    
+    feeSetting:
+      type: object
+      properties: 
+        rateType:
+          $ref: "#/components/schemas/rateType"
+        feeTrigger:
+          $ref: "#/components/schemas/feeTrigger"
+        feeReferenceAmount:
+          $ref: "#/components/schemas/feeReferenceAmount"
+        dayCountConvention:
+          $ref: "#/components/schemas/dayCountConvention"
+        percentageValue:
+          type: number
+          description: "Percentage of the fee"
+        billingMethod:
+          $ref: "#/components/schemas/billingMethod"
+
+    rateType:
+      type: string
+      enum:
+        - flat
+        - variable
+      description: |
+        Rate type of a fee.
+        Possible values are
+        * flat -> the rate of the fee remains constant.
+        * variable -> the rate of the fee can change.
+    
+    feeTrigger:
+      type: string
+      enum: 
+        - 'monthly-until-paid'
+        - 'on-financing'
+        - 'on-payment'
+      description: |
+        When the fee is calculated.
+        Possible values are
+        * monthly-until-paid -> the fee is calculated monthly until the invoice is paid.
```

Comment:
Maybe not calculated, more like `the fee will be capitalised every month until the invoice is paid`